backend/yandex/README.md

```diff
@@ -0,0 +1,32 @@
+# Wikimedia documentation
+
+## User guide
+
+```Python
+import json
+from yandex import spider
+data_list = spider(10)
+for data in data_list:
+    print(json.dumps(data, indent=2))
+```
+
+
+## Module documentation
+
+### wikimedia_crawl
```

Comment:
Please rename section.

Reply:
Can you give me some advise, first time writing documentation and i was following Long example

Reply:
> Can you give me some advise, first time writing documentation and i was following Long example

Following Long's doc is good enough for you.